src/BCMathExtended/BC.php

```diff
@@ -208,28 +152,18 @@ public static function mul($leftOperand, $rightOperand, $scale = null)
         return bcmul($leftOperand, $rightOperand, $scale);
     }
 
-    /**
-     * @param string $arg
-     * @return string
-     */
-    public static function exp($arg)
+    public static function exp(string $arg): string
     {
         $scale = self::DEFAULT_SCALE;
         $result = '1';
         for ($i = 299; $i > 0; $i--) {
-            $result = self::add(self::mul(self::div($result, $i, $scale), $arg, $scale), 1, $scale);
+            $result = self::add(self::mul(self::div($result, (string)$i, $scale), $arg, $scale), (string)1, $scale);
```

Comment:
Why don't we use the `'1'` directly?

Reply:
we are using  in `self::div($result, (string)$i, $scale)`

Reply:
@krowinski, I know that.

What about this: ` (string)1, $scale)`?

Reply:
but it's in loop and $results is changing every time

Reply:
The `1` is const value, I'm confused why we don't use the `1` directly :confused:.

Reply:
Ok I see, misunderstand you I thought you are talking about   this  $result = '1'; one. 

sry :)